inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/event/RecommenderState.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2019
+ * Ubiquitous Knowledge Processing (UKP) Lab
+ * Technische Universität Darmstadt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.recommendation.event;
+
+public enum RecommenderState
+{
+    TRAINING_STARTED, TRAINING_FINISHED, EVALUATION_STARTED, EVALUATION_FINISHED,
+    PREDICTION_STARTED, PREDICTION_FINISHED;
+    
+    @Override
+    public String toString()
```

Comment:
Why override `toString()` here? On a quick scan, I didn't see this value being used in the UI. If it is used in the UI, there are usually better (internationalizable) ways of mapping enum values to human-readable values.

Reply:
You are right, I guess I thought I would need it later and then didn't. Out of curiosity, can you point me to a better approach of mapping enums to descriptions?

Reply:
For Wicket Choice components, there is the EnumChoiceRenderer. For Labels, one could use a StringResourceModel.